README.md

```diff
@@ -5,21 +5,35 @@ Features:
 - Docker
 - REST APIs
 
-
-
+<br>
 
 ## Requirements
 - docker
+- uv
 
+<br>
 
 ## How to use
 First, if you are lazy execute
 ```
 source aliases.sh # or . aliases.sh
 ```
-This will create some shortcuts to run docker commands.
+This will create some shortcuts to run useful commands.
+
+> Private aliases is also possible (this file is gitignored).
+> To set up, run this. These must also be sourced manually.
+> ```
+> cp aliases-private.example.sh aliases-private.sh
+> ```
+> Some startup alias is recommended, for example (in your shell config):
```

Comment:
Isn't too much to have aliases to set other aliases? What about adding a line to check if `aliases-private.sh` is available and just source it within `aliases.sh`?